apps/flo-cli/src/cli/worktrees/create.worktrees.command.ts

```diff
@@ -140,9 +27,44 @@ export const createWorktreeCommand = new Command()
         `--dir, --directory <path>`,
         'create the worktree in a custom directory (will take precedence over --use-branch)',
     )
-    // @TODO: @floydnant there should be a `skip-hooks` option
+    .option(`-S, --skip-hooks`, 'skip running workflow hooks', false)
```

Comment:
Lets use the shared option and change `-s, --subDir` to sth else + make it shared and perhaps use in all other places too